src/utils/getSortedGoods.ts

```diff
@@ -0,0 +1,36 @@
+import { SortType } from '../types/SortType';
+
+interface SortingFilters {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+export function getSortedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortingFilters,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods = preparedGoods.sort(
+      (currentGood: string, nextGood: string) => {
+        switch (sortField) {
+          case SortType.ALPHABETICALLY:
+            return currentGood.localeCompare(nextGood);
+
+          case SortType.LENGTH:
+            return currentGood.length - nextGood.length;
+
+          default:
+            return 0;
+        }
+      },
+    );
+  }
```

Comment:
It's good that you're checking if `sortField` is provided before sorting. However, according to the checklist [CODE KNOWLEDGE], you should avoid direct DOM interactions and use React for rendering. Ensure that any sorting logic is decoupled from direct DOM manipulations and is used within React's rendering lifecycle when implementing this utility in a component.